bundles/org.openhab.binding.telegram/src/main/java/org/openhab/binding/telegram/internal/action/TelegramActions.java

```diff
@@ -636,6 +670,14 @@ public static boolean sendTelegramAnswer(ThingActions actions, @Nullable String
         return ((TelegramActions) actions).sendTelegramAnswer(replyId, message);
     }
 
+    public static boolean deleteTelegramQuery(ThingActions actions, @Nullable String replyId) {
+        if (actions instanceof TelegramActions) {
+            return ((TelegramActions) actions).deleteTelegramQuery(replyId);
```

Comment:
```suggestion
        if (actions instanceof TelegramActions telegramActions) {
            return telegramActions.deleteTelegramQuery(replyId);
```

Reply:
I made the same changes in my local 4.1.1 branch, and there it shows an error: telegramActions cannot be resolved
This is valid in my point of view, but why does it work in the main branch? 🤔